.gitignore

```diff
@@ -34,6 +35,9 @@ GitHub.sublime-settings
 .tags_sorted_by_file
 .vagrant
 .DS_Store
+.vscode
+.elixir_ls
+.sonarlint
```

Comment:
Let's delete all of editor config ignores. Instead, developers should add their specific ignores to a global gitignore file.